test/tools/runner/config.ts

```diff
@@ -358,6 +358,11 @@ export class TestConfiguration {
         key: localKey
```

Comment:
Can we improve this function while we're here, its called in one place from what I can see and its always passed 'local', and 96 random bytes.

On line 76 of driver.test.js:
```js
const kmsProviders = this.configuration.kmsProviders('local', crypto.randomBytes(96));
```

But also I don't see any usage with this taking in the new `'kmip'` option, do we need this change?

Reply:
Reefactored just to take the local key and return the object with it.